src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature

```diff
@@ -126,185 +126,309 @@ Scenario: Catalogue Solutions - edit price screen - Flat price with variable ord
 	When the User chooses to save
 	Then the Catalogue Solution is saved
 	And the Catalogue Solution dashboard is presented
 	And the Catalogue Solution is saved in the DB
+	Examples:
+	| ProvisioningType |
+	| declarative      |
+	| variable         |
+	| per patient      | 
+
 @ignore
-Scenario: Catalogue Solutions - edit price screen - Flat price Delivery date is more than 183 weeks after commencement date
-	Given the User is presented with the Catalogue Solution edit form for a variable flat price
+Scenario Outline: Catalogue Solutions - edit price screen - Delivery date validation - date is more than 183 weeks after commencement date
+	Given the User is presented with the Catalogue Solution edit form for a <ProvisioningType> flat price
 	And fills in the Catalogue Solution edit form with valid data
 	And the User enters a Delivery Date that is more than 183 weeks after the Commencement Date
 	When the User chooses to save
 	Then the Catalogue Solution is not saved
 	And the reason is displayed
+	Examples:
+	| ProvisioningType |
+	| declarative      |
+	| variable         |
+	| per patient      | 
+
 @ignore
-Scenario: Catalogue Solutions - edit price screen - Flat price Delivery Date cannot be before commencement date
-	Given the User is presented with the Catalogue Solution edit form for a variable flat price
+Scenario Outline: Catalogue Solutions - edit price screen - Delivery date validation - date cannot be before commencement date
+	Given the User is presented with the Catalogue Solution edit form for a <ProvisioningType> flat price
 	And fills in the Catalogue Solution edit form with valid data
 	And the User enters a Delivery Date that is before the Commencement Date
 	When the User chooses to save
 	Then the Catalogue Solution is not saved
 	And the reason is displayed
+	Examples:
+	| ProvisioningType |
+	| declarative      |
+	| variable         |
+	| per patient      | 
 
-Scenario: Catalogue Solutions - edit price screen - Flat price All data are valid
-	Given the User is presented with the Catalogue Solution edit form for a variable flat price
+Scenario Outline: Catalogue Solutions - edit price screen - All data are valid
+	Given the User is presented with the Catalogue Solution edit form for a <ProvisioningType> flat price
 	And fills in the Catalogue Solution edit form with valid data
 	When the User chooses to save
 	Then the Catalogue Solution is saved
 	And the Catalogue Solution dashboard is presented
 	And the Catalogue Solution is saved in the DB
+	Examples:
+	| ProvisioningType |
+	| declarative      |
+	| variable         |
+	| per patient      | 
 
-Scenario: Catalogue Solutions - edit price screen - Flat price Price is displayed to a minimum of 2 decimal places 
-	Given the User is presented with the Catalogue Solution edit form for a variable flat price
+Scenario Outline: Catalogue Solutions - edit price screen - Price is displayed to a minimum of 2 decimal places 
```

Comment:
i think at present most of these price scenarios can't run yet, can you ignore all the ones where the screen isn't reachable or the save isnt implemented